airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -75,6 +84,20 @@ public class ServerApp implements ServerRunnable {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ServerApp.class);
   private static final int PORT = 8001;
+  private static final String DRIVER_CLASS_NAME = "org.postgresql.Driver";
+
+  private static Configs configs;
+  private static DataSource configsDataSource;
+  private static DataSource jobsDataSource;
+  private static DSLContext configsDslContext;
+  private static DSLContext jobsDslContext;
```

Comment:
Just curious, is there a reason why we are adding these DataSource and DSLContext variables as `static` variables, as opposed to non-static member variables that are passed into the class during construction?

Reply:
This has been re-worked.